search/templates/cosmos/searchresults.html

```diff
@@ -23,4 +23,22 @@ <h4 class="card-title">
         </div>
     </div>
 </div>
+<br> {% endfor %}
+<h1>Recommended results</h1>
```

Comment:
You may use the `h5` tag instead of `h1` tags to make the text size looks proportional.